src/screens/CreatePost/index.js

```diff
@@ -106,7 +107,7 @@ const CreatePost = () => {
   const [dataImage, setDataImage] = React.useState([]);
   const [loading, setLoading] = React.useState(false);
   const [loadingPost, setLoadingPost] = React.useState(false);
-  const [dataProfile, setDataProfile] = React.useState({});
+  const [profile] = React.useContext(Context).profile;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from using a local state for `dataProfile` to using a context for `profile` is a significant one. This could potentially improve performance by avoiding unnecessary re-renders, but it also introduces a tighter coupling between this component and the context provider. Make sure that the context provider is set up correctly and that it wraps this component in the component tree.
```diff
- const [dataProfile, setDataProfile] = React.useState({});
+ const [profile] = React.useContext(Context).profile;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->